javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,10 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var output = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```

Comment:
This function is meant to filter out strings, not gather all numbers. What if the argument array included other data types like `null` or booleans?